client/src/features/home/NeedsAndOffers.tsx

```diff
@@ -65,9 +65,17 @@ function NeedsAndOffers(props: Props): JSX.Element {
         {assets.map((asset) => (
           <NavLink to={`/asset/${asset.id}`} key={asset.id} className={classes.card}>
             <Card variant="outlined">
-              <img src={asset.imgUrls[0]} className={classes.cardImg} alt={asset.title} />
+              <img
+                src={
+                  asset.imgUrls
```

Comment:
right idea, but i think it's probably better to do this a little higher up
`{asset.imgUrls.length && (<img ...`
then we don't need the fallback img you hardcoded here

Reply:
When you say to do this higher up, you mean in the parent component? I'm not sure what your code snippet with `asset.imgUrls.length` is hinting at.